x-pack/plugin/transform/qa/multi-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/transform/integration/TransformChainIT.java

```diff
@@ -85,15 +71,67 @@ public class TransformChainIT extends TransformRestTestCase {
           },
           "settings": {
             "unattended": true,
-            "deduce_mappings": %s
+            "deduce_mappings": %s,
+            "use_point_in_time": %s
           }
         }""";
 
     private TestThreadPool threadPool;
 
     @Before
-    public void createThreadPool() {
+    public void createThreadPool() throws IOException {
```

Comment:
I think this method should be renamed to something like `setup`, or, if that clashes with a base class method, then `setupTransformTests` or something like that. It does much more than create a thread pool now.

Reply:
Done.